graylog2-web-interface/src/views/components/views/ViewHeader.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import React, { useCallback, useMemo, useState } from 'react';
+import styled, { css } from 'styled-components';
+
+import { useStore } from 'stores/connect';
+import { ViewStore } from 'views/stores/ViewStore';
+import { Icon } from 'components/common';
+import { Row } from 'components/bootstrap';
+import ViewPropertiesModal from 'views/components/views/DashboardPropertiesModal';
+import onSaveView from 'views/logic/views/OnSaveViewAction';
+
+const StyledList = styled.dl(() => css`
+  display: flex;
+  gap: 8px;
+  margin: 0;
+  flex-wrap: wrap;
+  align-items: center;
+  dd {
+    font-style: italic;
+  }
+  dt {
+    text-transform: capitalize;
+  }
+`);
+
+const Content = styled.div(({ theme }) => css`
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  align-items: center;
+  padding-bottom: ${theme.spacings.sm}
+`);
+const EditButton = styled.div(() => css`
+  width: 25px;
+`);
+
+const ViewHeader = () => {
+  const { view } = useStore(ViewStore);
+  const isSavedView = useMemo(() => view?.id && view?.title, [view]);
+  const [showMetadataEdit, setShowMetadataEdit] = useState<boolean>(false);
+  const toggleMetadataEdit = useCallback(() => setShowMetadataEdit((cur) => !cur), [setShowMetadataEdit]);
+
+  return isSavedView ? (
+    <Row>
+      <Content>
+        <StyledList>
+          <dt>{view.type.toLocaleLowerCase()}:</dt>
+          <dd>{view.title}</dd>
+        </StyledList>
```

Comment:
The breadcrumb works as expected, just a few thoughts
- I would not display the edit icon in top right corner, it look like it opens a mode where you can edit for example the widget layout.
- Let's call the view type "Dashboards" instead of "Dashboard"
- We can consider making "Dashboards" a link. The link to the dashboards page is not far away, but it is still a breadcrumb and I can imagine some users will expect a link.
- Currently the bold text feels heavy
<img width="671" alt="image" src="https://user-images.githubusercontent.com/46300478/191050990-dc326c3b-db58-4b36-9e09-c966d41433c0.png">

we can probably define a more subtle style
<img width="685" alt="image" src="https://user-images.githubusercontent.com/46300478/191050834-41199dba-c08f-40da-b002-acac9b1c4b02.png">